lib/string_reverse.rb

```diff
@@ -1,4 +1,16 @@
 # A method to reverse a string in place.
 def string_reverse(my_string)
-  my_string << "not implemented"
+  #my_string << "not implemented"
+    index = 0
+    while my_string != nil && my_string.length != 0 && index < (my_string.length/2)
+      length = my_string.length

```

Comment:
You could move this outside the loop and avoid repeated calls to my_string.length method since the length of the input parameter will not be changing through each iteration of the while loop. Also, that way, you can use the variable in your loop's condition.

Reply:
Thanks for the suggestions Shruti!